.vscode/launch.json

```diff
@@ -0,0 +1,21 @@
+{
+    // Use IntelliSense to learn about possible Node.js debug attributes.
+    // Hover to view descriptions of existing attributes.
+    // For more information, visit: https://go.microsoft.com/fwlink/?linkid=830387
+    "version": "0.2.0",
```

Comment:
We usually exclude IDE settings from git, you should add `.vscode` to your global `~/.gitignore` file.